rest-api-spec/src/main/resources/rest-api-spec/test/search_shards/10_basic.yaml

```diff
@@ -14,7 +14,7 @@
 ---
 "Search shards aliases with and without filters":
   - skip:
-      version: " - 6.0.0" # temporarily disabled
+      version: " - 5.4.1" # bugs fixed in 5.4.1
       reason: indices section was added in 5.1.0
```

Comment:
the reason specified below the version mentions 5.1.0 which is not in sync with the version anymore. We should update the reason I think.